### ISSUE TITLE
Is there any way to emulate lcov's baseline functionality?

### ISSUE THREAD

**Issue by PhilipMourdjis** [RPGillespie6/fastcov#28]
I have found that fastcov does not include some files in its output that should show up as 0% coverage.

lcov has a function that captures an initial view of the code using the -i flag, this can then be combined with the result of a test run to get a more accurate view of completely untested files.

I am dealing with a very large project and using lcov even for just the baseline is taking too long. Is there any way to get fastcov to do what i want? Would this be a reasonable feature request?

**Comment by RPGillespie6** [comment]
fastcov is currently only processing .gcda files to generate coverage, which means that if the gcc instrumentation was never run (even by an empty file that does nothing), it will not show up in the final report. I suppose it could search for .gcno files generated by the compiler and generate a 0% coverage entry in the report for sources files that have an accompanying gcno. If so, I don't think that would be too difficult to add, let me look into it.

**Comment by RPGillespie6** [comment]
Ok, after researching a bit, it looks like adding support for this feature will be trivial. We can take advantage of a feature of gcov to process gcno and gcda at the same time if a flag is passed. If you want to quickly try it, change `fastcov.py:88` to search for `**/*.gcno` instead of `**/*.gcda` and it should produce the desired result.

I just need to create a new flag, add a functional test for it, should be done today or tomorrow.

**Comment by philip-mourdjis-oxb** [comment]
Sounds great! I'll give it a go.

**Comment by PhilipMourdjis** [comment]
Working well for me, any chance the pip package can be updated with this?

**Comment by RPGillespie6** [comment]
Yes, I will bump the version and tag a new release later today which should update PyPI to 1.3.

**Comment by RPGillespie6** [comment]
@PhilipMourdjis latest pip package should now have this change picked up.
